cmd/gdsutil/main.go

```diff
@@ -58,8 +57,8 @@ func main() {
 					EnvVar: "GDS_DATABASE_URL",
 				},
 				cli.BoolFlag{
-					Name:  "s, stringify",
-					Usage: "stringify keys otherwise they are base64 encoded",
+					Name:  "b, b64encode",
+					Usage: "base64 encode keys (otherwise they will be utf-8 decoded)",
```

Comment:
Figured the common case was stringify, so mind as well make that the default!

Reply:
Nice!